content/13.conclusion.md

```diff
@@ -1,17 +1,28 @@
-## Ethics {#ethics}
+## Concluding remarks and ethics-related considerations {#conclusion}
+While DL continues to be a powerful, transformative tool within life sciences research—spanning basic biology and pre-clinical science to varied translational approaches and clinical studies—it is important to comment on some ethics-related considerations. 
+For instance, despite that DL methods are helping to increase medical efficiency through improved diagnostic capability and risk assessment, certain biases may be inadvertently introduced into models related to patient age, race, and gender [@doi:10.1002/hast.977]; as previously mentioned, DL practitioners may make use of datasets lacking diverse populations and patient characteristics [@doi:10.1377/hlthaff.2014.0048], thereby contributing to these problems (please refer to Tip 4).
```

Comment:
Thank you for re-wording this text. I think its greatly improved since my rough draft!